docs/FAQ.md

```diff
@@ -56,7 +56,7 @@ No
 
 ### What is the policy on supporting end-of-life(EOL) Ruby versions?
 
-We will target the latest EOL Ruby version, please check the link below the see the current EOL version.
+We will target all non EOL Ruby versions, please check the link below the see the current EOL version.
```

Comment:
to clarify, are intending to say that we don't support any EOL versions? The original language says 1 EOL, this says no EOL, so just want to be clear. Historically we've targeted one EOL to not completely shut out folks who may be on a weird platform that doesn't have the latest and greatest ruby available. Most folks run omnibus installs so it's not a problem but there are some weird platforms out there. For ex on raspi where there's no sensu omnibus available.